pkg/apis/kubeone/v1beta1/conversion.go

```diff
@@ -30,3 +30,21 @@ func Convert_v1beta1_Features_To_kubeone_Features(in *Features, out *kubeone.Fea
 	// The PodPresets field has been dropped from v1beta2 API.
 	return nil
 }
+
+// Convert_v1beta1_CloudProviderSpec_To_kubeone_CloudProviderSpec is an autogenerated conversion function.
+func Convert_v1beta1_CloudProviderSpec_To_kubeone_CloudProviderSpec(in *CloudProviderSpec, out *kubeone.CloudProviderSpec, s conversion.Scope) error {
+	if err := autoConvert_v1beta1_CloudProviderSpec_To_kubeone_CloudProviderSpec(in, out, s); err != nil {
+		return err
+	}
+	// PacketSpec has been renamed to EquinixMetalSpec
```

Comment:
You are supposed to write a MANUAL conversion here, since automatic didn't worked. It mean you have to copy from PacketSpec type to EquinixMetalSpec

Reply:
Let's discuss conversions tomorrow. IMO, conversions should be in-place only if we support migration from Packet to Equinix Metal. Otherwise, those two are not the same and shouldn't be converted.

Reply:
Good point. Then it should error out (since there would be no migration).